### ISSUE TITLE
Cannot deploy when framework packages of a higher version are already installed

### ISSUE THREAD

**Issue by NickGerleman** [microsoft/react-native-windows#5482]
Likely regression from #5454

```
 Add-AppXPackage : Deployment failed with HRESULT: 0x80073D06, The package could not be installed because a higher
version of this package is already installed.

Windows cannot install package Microsoft.UI.Xaml.2.3_2.31911.29002.0_x86__8wekyb3d8bbwe because it has version
2.31911.29002.0. A higher version 2.32002.13001.0 of this package is already installed.

NOTE: For additional information, look for [ActivityId] 0ff47368-562e-000e-1ce0-f40f2e56d601 in the Event Log or use
the command line Get-AppPackageLog -ActivityID 0ff47368-562e-000e-1ce0-f40f2e56d601

At line:1 char:7
+ try { Add-AppXPackage C:\Users\nickg\Desktop\react-native-windows\pac ...
+       ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (C:\Users\nickg\...I.Xaml.2.3.appx:String) [Add-AppxPackage], Exception
    + FullyQualifiedErrorId : DeploymentError,Microsoft.Windows.Appx.PackageManager.Commands.AddAppxPackageCommand

× Installing dependent framework packages
 × Failed to deploy: Installing dependent framework packages
```

**Comment by elcssmouhsine** [comment]
i am getting similar error  Add-AppxPackage : Deployment failed with HRESULT: 0x80073CF3, Package failed updates, dependency or conflict validation.
Windows cannot install package ae76a144-058d-41e4-8c81-f9bbb6297b42_1.0.0.0_x86__ad8pwfkyh69vj because this package depends on a framework that could not be found. Provide the
framework "Microsoft.UI.Xaml.2.3" published by "CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", with neutral or x86 processor architecture
and minimum version 2.31911.29002.0, along with this package to install. The frameworks with name "Microsoft.UI.Xaml.2.3" currently installed are: {}
NOTE: For additional information, look for [ActivityId] 3b5077ec-5c0e-0004-213b-513b0e5cd601 in the Event Log or use the command line Get-AppPackageLog -ActivityID
3b5077ec-5c0e-0004-213b-513b0e5cd601
At C:\Users\elcss\Documents\examappnative\node_modules\react-native-windows\local-cli\runWindows\utils\WindowsStoreAppUtils.ps1:183 char:5
+     Add-AppxPackage -Path $Path -Register -ForceApplicationShutdown
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (C:\Users\elcss\...ppxManifest.xml:String) [Add-AppxPackage], IOException
    + FullyQualifiedErrorId : DeploymentError,Microsoft.Windows.Appx.PackageManager.Commands.AddAppxPackageCommand

**Comment by asklar** [comment]
Also need #5543

**Comment by Brouilles** [comment]
Same problem here.

**Comment by goroya** [comment]
me too.

**Comment by chrisglein** [comment]
Fix has been ported to 0.62 very recently, so you should stop seeing this with the next release of that (next week).
